python/paddle/v2/fluid/tests/book/test_label_semantic_roles.py

```diff
@@ -211,12 +211,74 @@ def main():
                 if batch_id != 0:
                     print("second per batch: " + str((time.time() - start_time)
                                                      / batch_id))
-
-            # exit early for CI
-            exit(0)
+                if save_dirname is None:
+                    exit(0)
+                # Set the threshold low to speed up the CI test
+                if float(pass_precision) > 0.1:
+                    fluid.io.save_inference_model(save_dirname, [
+                        'word_data', 'verb_data', 'ctx_n2_data', 'ctx_n1_data',
+                        'ctx_0_data', 'ctx_p1_data', 'ctx_p2_data', 'mark_data'
+                    ], [feature_out], exe)
+                    return
```

Comment:
For the first point, I will change the code to 
```python
if float(pass_precision) > 0.1:
    if save_dirname is not None:
        fluid.io.save_inference_model(save_dirname, [
                        'word_data', 'verb_data', 'ctx_n2_data', 'ctx_n1_data',
                        'ctx_0_data', 'ctx_p1_data', 'ctx_p2_data', 'mark_data'
                    ], [feature_out], exe)
        return
    else:
        # exit early for CI test
        exit()
```
We still need exit() because 1) we want to make the CI test time short; 2) If we don't have a save_dirname, we don't have a model saved, then we should use exit() to terminal python program instead of keep running infer method which will raise a error because the model and parameters haven't been created by the train method.  What do you think @Xreki?

For the second point,  I agree, I will change the output to crf_decoding.

Reply:
How about change to:
```python
# Set the threshold low to speed up the CI test
if float(pass_precision) > 0.1:
    if save_dirname is not None:
        fluid.io.save_inference_model(save_dirname, [
                        'word_data', 'verb_data', 'ctx_n2_data', 'ctx_n1_data',
                        'ctx_0_data', 'ctx_p1_data', 'ctx_p2_data', 'mark_data'
                    ], [feature_out], exe)
    return
```

In my code, the `return` is in the wrong position. That is, the function will return wether `save_dirname` is `None` or not.

Reply:
Nice! Done

Reply:
When I change the fetch list from [feather_out] to [crf_decode], the test gives the following error:  

```
348: [libprotobuf ERROR /paddle/build/third_party/protobuf/src/extern_protobuf/src/google/protobuf/message_lite.cc:119] Can't parse message of type "paddle.framework.proto.ProgramDesc" because it is missing required fields: (cannot determine missing fields for lite message)
348: Traceback (most recent call last):
348:   File "test_label_semantic_roles.py", line 293, in <module>
348:     infer(save_dirname)
348:   File "test_label_semantic_roles.py", line 248, in infer
348:     fetch_targets] = fluid.io.load_inference_model(save_dirname, exe)
348:   File "/paddle/build/python/build/lib-python/paddle/v2/fluid/io.py", line 391, in load_inference_model
348:     program = Program.parse_from_string(program_desc_str)
348:   File "/paddle/build/python/build/lib-python/paddle/v2/fluid/framework.py", line 901, in parse_from_string
348:     p.desc = core.ProgramDesc(binary_str)
348: paddle.v2.fluid.core.EnforceNotMet: Fail to parse program_desc from binary string. at [/paddle/paddle/framework/program_desc.cc:62]
```

The reason is probably because of this TODO item:
```
    # TODO(qiao)
    # add dependency track and move this config before optimizer
    crf_decode = fluid.layers.crf_decoding(
        input=feature_out, param_attr=fluid.ParamAttr(name='crfw'))
```
I think I will stick to [feature_out] in this PR and figure this out in future pr.